doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1347,41 +1429,64 @@ To delete a timestamping service, follow these steps.
 
 _To change the security server’s internal TLS key and certificate_, follow these steps.
 
-1. On the **Configuration** menu, select **System Parameters**. The system parameters view is opened.
-2. In the **Internal TLS Certificate** section, click **Generate New TLS Key** and in the window that opens, click **Confirm**.
+1. On the **Navigation tabs**, select **Keys and Certificates
+
+2. In the opening view, select **Security Server TLS Key** tab
+
+3. In the opening view, click **GENERATE KEY** and in the dialog that opens, click **CONFIRM**.
 
    The security server generates a key used for communication with the client information systems, and the corresponding self-signed certificate. The security server's certificate fingerprint will also change. The security server's domain name is saved to the certificate's *Common Name* field, and the internal IP address to the *subjectAltName* extension field.
```

Comment:
Not changed in this PR, but here is yet another way to highlight field names (italic)

> The security server's domain name is saved to the certificate's *Common Name* field, and the internal IP address to the *subjectAltName* extension field.

Reply:
Changed to *bolded*